Description: Update autotools
 In particular, avoid the following warning:
 configure.ac:11: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms
 are deprecated.  For more info, see:
 configure.ac:11: http://www.gnu.org/software/automake/manual/automake.html
 #Modernize-AM_005fINIT_005fAUTOMAKE-invocation
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-11-05

--- a/configure.ac
+++ b/configure.ac
@@ -8,7 +8,7 @@
 
 AC_INIT(WMBiff, 0.4.27, wmbiff-devel@lists.sourceforge.net, wmbiff)
 AC_CONFIG_AUX_DIR(autoconf)
-AM_INIT_AUTOMAKE(wmbiff, 0.4.27) dnl tarded syntax for automake <= 1.5
+AM_INIT_AUTOMAKE
 AC_CONFIG_HEADER([config.h])
 dnl make sure autoheader finds version, implicitly defined above.
 AH_TEMPLATE([VERSION], [wmbiff's release version])
